api/v01/entities/vrp_input.rb

```diff
@@ -0,0 +1,521 @@
+      use :vrp_request_shipment
+    end
+    at_least_one_of :services, :shipments
+
+    optional(:relations, type: Array, desc: 'Not available with periodic heuristic') do use :vrp_request_relation end
+
+    optional(:subtours, type: Array, desc: 'Not available with periodic heuristic') do use :vrp_request_subtour end
+
+    optional(:routes, type: Array, desc: '') do use :vrp_request_route end
+
+    optional(:configuration, type: Hash, documentation: { desc: 'Describe the limitations of the solve in term of computation' }) do
+      use :vrp_request_configuration
+    end
+  end
+
+  # Input as expected in CSV
+  params :request_files do # rubocop:disable Metrics/BlockLength
+                           # We expect here to keep the definition of the high level model in a single place
+    optional :points, type: Array,
+                      documentation: { hidden: true, desc: 'Warning : CSV Format expected here ! Particular place in the map' },
+                      coerce_with: ->(path) { path.is_a?(Array) ?path : Api::V01::CSVParser.call(File.open(path[:tempfile], 'r:bom|utf-8').read, nil) } do
+      use :vrp_request_point
+    end
+
+    optional :units, type: Array,
+                     documentation: { hidden: true, format: 'CSV', desc: 'Warning : CSV Format expected here ! The name of a Capacity/Quantity' },
+                     coerce_with: ->(path) { path.is_a?(Array) ?path : Api::V01::CSVParser.call(File.open(path[:tempfile], 'r:bom|utf-8').read, nil) } do
+      use :vrp_request_unit
+    end
+
+    optional :timewindows, type: Array,
+                           documentation: { hidden: true, format: 'CSV', desc: 'Warning : CSV Format expected here ! Time slot while the activity may be performed' },
+                           coerce_with: ->(path) { path.is_a?(Array) ?path : Api::V01::CSVParser.call(File.open(path[:tempfile], 'r:bom|utf-8').read, nil) } do
+      use :vrp_request_timewindow
+    end
+
+    optional :capacities, type: Array,
+                          documentation: { hidden: true, format: 'CSV', desc: 'Warning : CSV Format expected here ! Define the limit of entities the vehicle could carry' },
+                          coerce_with: ->(path) { path.is_a?(Array) ?path : Api::V01::CSVParser.call(File.open(path[:tempfile], 'r:bom|utf-8').read, nil) } do
+      use :vrp_request_capacity
+    end
+
+    optional :quantities, type: Array,
+                          documentation: { hidden: true, format: 'CSV', desc: 'Warning : CSV Format expected here ! Define the entities which are taken or dropped' },
+                        coerce_with: ->(path) { path.is_a?(Array) ?path : Api::V01::CSVParser.call(File.open(path[:tempfile], 'r:bom|utf-8').read, nil) } do
+      use :vrp_request_quantity
+    end
+
+    optional :services, type: Array,
+                        documentation: { hidden: true, format: 'CSV', desc: 'Warning : CSV Format expected here ! Independent activity, which does not require a context' },
```

Comment:
Good practice suggest that there should be a dot if there are at least two sentences in the description.

```suggestion
                        documentation: { hidden: true, format: 'CSV', desc: 'Warning : CSV Format expected here ! Independent activity, which does not require a context.' },
```